samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -72,21 +68,23 @@ private void configureSdk(CustomerIOBuilder builder, final CustomerIOSDKConfig s
             builder.cdnHost(cdnHost);
         }
 
-        if (sdkConfig.getFlushAt() != null) {
-            builder.flushAt(sdkConfig.getFlushAt());
-        }
-        if (sdkConfig.getFlushInterval() != null) {
-            builder.flushInterval(sdkConfig.getFlushInterval());
+        if (sdkConfig.isTestModeEnabled()) {
+            builder.flushInterval(1);
+            builder.flushAt(1);
```

Comment:
Have we previously decided on these values? Flushing every 1 second seems too aggressive. Given that `flushAt` is already set to `1`, the SDK will likely flush after each event. So setting `flushInterval` to `1` second seems unnecessary. I suggest increasing the `flushInterval` to `5` or preferably `10` seconds to avoid the SDK repeatedly checking for events every second without reason.

Reply:
You are absolutely right, that makes sense. Maybe we can leave the `flushInterval` at default and only depend on `flushAt` to send events immediately as they happen.